matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/offers/MatchedBet.java

```diff
@@ -0,0 +1,92 @@
+package com.matchbook.sdk.core.clients.rest.dtos.offers;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+import com.matchbook.sdk.core.clients.rest.dtos.prices.Currency;
+import com.matchbook.sdk.core.clients.rest.dtos.prices.OddsType;
+
+public class MatchedBet {
```

Comment:
I've implemented a more completed models for offers in a dedicated branch. They include `MatchedBet`, `Offer` and `Position` you have included in this PR. I'll merge the code to fix conflicts after these changes have been moved to master.